src/static/css/pages/bureau.less

```diff
@@ -95,12 +95,12 @@
     &_image {
         width: 250px;
 
-        .respond-to-max(599px, {
+        .respond-to-max(@mobile-max, {
             width: 100%;
```

Comment:
Width is 100% by default. This could move up to L:96 and the 250px width could move to a media query for desktop size (although that 768px below is kinda odd so maybe not)